src/SourceGenerators/Uno.UI.Tasks/Content/Uno.UI.Tasks.targets

```diff
@@ -178,22 +180,72 @@
 	</Target>
 
 	<!--
-	  XAML Resources Trimming
+	  Upri Resources Trimming
+	-->
+
+	<Target Name="_UnoLinkerUpriSubstitutions"
+		Condition="'$(DesignTimeBuild)' != 'true' and '$(BuildingInsideUnoSourceGenerator)' == ''"
+		AfterTargets="UnoResourcesGeneration"
+		BeforeTargets="_UnoLinkerCollectSubstitutions">
+
+		<ItemGroup>
+			<_UnoUpriResource Condition="'%(GeneratedFiles.UnoResourceTarget)' == 'Uno'" Include="@(GeneratedFiles->'%(LogicalName)')">
+				<Language>%(GeneratedFiles.Language)</Language>
+			</_UnoUpriResource>
+		</ItemGroup>
+
+		<UpriSubstitutionsGeneratorTask_v0
+			AssemblyName="$(AssemblyName)"
+			Resources="@(_UnoUpriResource)"
+			OutputFile="$(IntermediateOutputPath)\Substitutions\UpriResources.Substitutions.xml" />
```

Comment:
Do we need `<FileWrites Include="..." />` here?

Reply:
Yes, it will avoid leaking files on clean